### PR TITLE
made results view prettier, now it has a list and results view

### PR DESCRIPTION
**Pull request by michaelmld**

### CODE REVIEW

**Comment by michaelmld** [comment]
screen shot 2017-04-11 at 10 02 59 pm

File: public/stylesheets/results.css [hunk 1]
```diff
@@ -1,3 +1,50 @@
 .resultColumn {
   text-align: center;
-}
\ No newline at end of file
+}
+
+.listContainer.col-md-3 {
+  height:100vh;
```
Comment (a6dang) [inline public/stylesheets/results.css]: what does the vh do?

**Comment by a6dang** [comment]
so on load will the first champion stats comparison be shown on the detail view?

**Comment by michaelmld** [comment]
on first load it shows nothing LOL , i got lazy anthony i dindt feel like doing that part. You have to click to display the stats

File: public/stylesheets/results.css (hunk 1 above)
Comment (michaelmld) [inline public/stylesheets/results.css]: vh (viewport height)

Comment (michaelmld) [inline public/stylesheets/results.css]: it should make the height dynamic so i dont have to set a fixed height, heigh:100% doesnt work http://stackoverflow.com/questions/1575141/make-div-100-height-of-browser-window

**Comment by michaelmld** [comment]
screen shot 2017-04-12 at 12 44 31 pm

File: views/results.ejs [hunk 2]
```diff
@@ -10,43 +10,82 @@
...
+<script>
+    $(document).ready(function() {
+        $(".listRow").click(function () {
+            var attr = $(this).attr('id');
+            $(".rightContainerContentPanel").hide();
+            $(".congratsView").hide();
+            $("#"+"result"+attr).toggle();
+        });
+    });
+</script>
```
Comment (tdashroy) [inline views/results.ejs]: how did you decide on putting this here vs. in a separate file?

File: views/results.ejs [hunk 3]
```diff
@@ -10,43 +10,82 @@
...
+
+    <div class="col-md-9 resultsContainer">
+
+        <div class="congratsView" >
+            <% if(winner > 0){ %>
+            <h1>Congrats <%= summonerA %> you are not trash! </h1>
+            <% } else if(winner < 0) { %>
+            <h1> Congrats <%= summonerB %> you are not trash! </h1>
+            <% }  else {%>
+            <h1> Tie </h1>
```
Comment (tdashroy) [inline views/results.ejs]: Suggest: "too bad, you're both trash"

File: views/results.ejs [hunk 4]
```diff
@@ -10,43 +10,82 @@
...
             <% } %>
         </div>
+        <% for(result of resultList) { %>
+        <div class="rightContainerContentPanel" id=<%="result"+result.championName%> >
+            <% for(thing of result.result) {%>
```
Comment (tdashroy) [inline views/results.ejs]: can we rename `thing`? it's not obvious to me what this is.

Comment (michaelmld) [inline views/results.ejs]: lol oops forgot i did that

**Pull request merged**